tests/components/logging/aggregate.test.js

```diff
@@ -182,6 +183,20 @@ describe('payloads', () => {
     loggingAggregate.ee.emit(LOGGING_EVENT_EMITTER_CHANNEL, [1234, Symbol('test'), {}, 'error'])
     expect(logs.pop().message).toEqual('Symbol(test)')
   })
+
+  test('initialLocation should be in pageUrl of log object attributes', async () => {
+    const log = new Log(
+      Math.floor(runtime.timeKeeper.correctAbsoluteTimestamp(
+        runtime.timeKeeper.convertRelativeTimestamp(1234)
+      )),
```

Comment:
For my understanding, why do we invoke timeKeeper methods here?  Could we have used just `1234` as the pretend timestamp?

Reply:
Yeah just use a random number will do instead of relying on these 2 timekeeper fns (side note: btw they have been replaced in src with correctRelativeTimestamp)

Reply:
^ Good point.

hey @ellisong, I'll leave it up to you whether to make a cleanup here or not.  I'm going to re-approve the PR since this is not technically blocking and I _think_ y'all resolved the other comment.